source/documentation/services/airflow/index.md

```diff
@@ -70,37 +76,51 @@ After your request is granted, you will be added to a GitHub team that will give
 
 1. Create a repository using one of the provided runtime templates:
 
-    > You can create this repository in either the [`ministryofjustice`](https://github.com/ministryofjustice/) or [`moj-analytical-services`](https://github.com/moj-analytical-services/) GitHub organisation
-    >
-    > Repository standards, such as branch protection, are out of scope for this guidance
-    >
-    > For more information on runtime templates, please refer to [runtime templates](#runtime-templates)
+- [Python Airflow Template](https://github.com/new?template_name=analytical-platform-airflow-python-template&template_owner=ministryofjustice)
```

Comment:
Bit more of a refactoring here to make the templates links stand out more. Markdown hates numbers, lists and code blocks so had to keep the numbering going using `2.\` etc